/*
 * Copyright 2015 The Kythe Authors. All rights reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

syntax = "proto3";

package kythe.proto;

option go_package = "kythe.io/kythe/proto/claim_go_proto";
option java_package = "com.google.devtools.kythe.proto";

import "kythe/proto/storage.proto";

// An assignment of analysis responsibility.
message ClaimAssignment {
  // The compilation responsible for analyzing the dependency.
  VName compilation_v_name = 1;
  // The dependency that needs analyzed.
  VName dependency_v_name = 2;
}
